client/extensions/woocommerce/app/promotions/promotion-form-type-card.js

```diff
@@ -34,6 +34,9 @@ const PromotionFormTypeCard = ( {
 } ) => {
 	const promotionType = ( promotion && promotion.type ? promotion.type : '' );
 
+	const productTypesDisabled = ( promotion.couponId );
```

Comment:
Did you want to Boolean these two lines?

Reply:
Yes I did, thanks!